### ISSUE TITLE
Task 8: Review the Introduction of the OpenSource

### ISSUE THREAD

**Issue by edovio** [edovio/OpenStudentBook#41]
Review the Introduction of the OpenSource 
(Deadline: 30 September)

Revisionare la sezione Introduttiva all'OpenSource 
(Deadline: 30 Settembre)

**Comment by FrancescaMancini** [comment]
@edovio Do you think that it might be good to expand this section and include the role of open source software in the reproducibility and replicability of research studies?

**Comment by edovio** [comment]
@FrancescaMancini I think could be interesting to have a more large scope of the open source movement :)

**Comment by amirad** [comment]
This video was recently produced that you could include! https://www.youtube.com/watch?v=7c0IrsDsNaw

**Comment by amirad** [comment]
I really like that there is an example! I thought Moodle is an example of an Open Project - is it? We should give an example of what is open source and an example for an open project! As well as define what the difference in those are. 

Abby did a great job with the open leadership training and laying examples out in there - maybe it will be helpful? https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/

**Comment by Mte90** [comment]
On my talk about What is open and and Linux (in Italian) I have a little list of open source project or open philoshpy projects that everyone know also if it is not a IT appassionate.
As reference: http://mte90.github.io/Presentazione-OSC/#/11

Moodle is an example but probably add something that the people know or use but not know all the details can be interesting. I saw that video about VLC yesterday https://www.youtube.com/watch?v=jWx1P93nS0c and show what I mean. The fact that people see your t-shirt an reconize the logo and appreciate what you do but you are not a open source fan. 
We are talking to students that can be interesting and myabe add example that they know can be more comfortable or interesting to them.

**Comment by edovio** [comment]
Yeah all that things are interesting, but considering that the guide could be distributed as a book (PDF, Ebook). To include Video or or other media things we can evaluate to create in this repo another section to create a little website with the content (including the Media content). 

When I talked about the openSource in my talk I used this video: https://www.youtube.com/watch?v=a8fHgx9mE5U made by blueprint:) It's super easy and understandable (and with the subtext localized in different language). 

We can define if could be helpful to create a little website for the OpenStudentBook to include other materials then with  a book could be given. 
About the OpenLeadershipTraining we can figure out to put a new chapter where we discuss about how opening you project, using a term as "Openify" or other similar, where we can illustrate in details more things. I done a talk about the OpenLeadershipTraining (http://slide.edovio.com/clsxItaly/#/). To start I can put some content from there an put some link (when necessary) at the awesome OpenLeadershipTraining done by Abby :)
